src/components/ToolbarMenu.js

```diff
@@ -40,7 +38,7 @@ export class ToolbarMenu extends React.Component {
         <svg width="14" height="8">
           <path fill="#666" d="M 5 1.5 L 14 1.5 L 9.5 7 z" />
         </svg>
-        <ul className="toolbar-menu-items" style={ulStyle}>
+        <ul className={'toolbar-menu-items' + (visible ? ' open' : '')}>
```

Comment:
Could use the `classnames` package for this, but I think it's good to keep the dependency footprint small, so this is good.